contributions/services/mpesa.py

```diff
@@ -0,0 +1,51 @@
+import os
+import requests
+from datetime import datetime
+from requests.auth import HTTPBasicAuth
+from django.utils import timezone
+
+
+class MpesaDarajaClient:
+    base_url        = "https://sandbox.safaricom.co.ke"
```

Comment:
This should be an environment variable since difference environments use different urls.